test/data/EPCISDocument-AggregationEvent.json

```diff
@@ -7,6 +7,8 @@
       {
         "type": "AggregationEvent",
         "eventTime": "2013-06-08T14:58:56.591Z",
+        "recordTime": "2005-04-05T02:33:31.116Z",
+        "eventID":"urn:epc:id:sgln:1234567.11111.400",
```

Comment:
An eventID should have this format:
ni:///sha-256;36abb3a2c0a726de32ac4beafd6b8bc4ba0b1d2de244490312e5cbec7b5ddece?ver=CBV2.0

And this is the same for every events you modified